model/fieldtypes/DBFloat.php

```diff
@@ -60,3 +60,7 @@ public function prepValueForDB($value) {
 	}
 
 }
+
+if (PHP_MAJOR_VERSION < 7) {
+	include_once 'compat/Float.php';
```

Comment:
I still don't like classes being manually included in this way. It's not the job of DBFloat to trigger loading of Float.php.